.github/workflows/spell-check.yml

```diff
@@ -0,0 +1,53 @@
+name: Spell Check
+
+on:
+  # Run on-demand using workflow_dispatch
+  workflow_dispatch:
+  # Schedule to run once a week
+  schedule:
+    - cron: '0 0 * * 1' # Runs every Monday at midnight UTC
+
+jobs:
+  spelling:
+    name: Spell check
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@v3 # Using a specific version of checkout
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.x'
+
+      - name: Install spell checker
+        run: |
+          pip install check-spelling==0.0.20 # Use a specific version of the spell checker
+
+      - name: Run spell checker
+        uses: check-spelling/check-spelling@0.0.19 # Pin the check-spelling action to a specific version
```

Comment:
I tested the workflow in my fork and got this error (https://github.com/sm1990/ballerina-dev-website/actions/runs/11267541453/job/31332834532). Also it seems the latest version is `v0.0.22` (https://github.com/marketplace/actions/check-spelling)

Reply:
@sm1990  can you try again?

Reply:
I'm still getting the same error:- https://github.com/sm1990/ballerina-dev-website/actions/runs/11271737213/job/31345188013#step:1:36

Reply:
@sm1990 can you try this again ?